aten/src/ATen/Dispatch.h

```diff
@@ -416,6 +416,46 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
     }                                                                       \
   }()
 
+#define AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES_AND3(                          \
```

Comment:
One of the ways to measure if the complex half support is stable could be to give complex half first class status and include it in complex types by default and ensure everything works